plugins/vswitch/vswitch.cpp

```diff
@@ -77,6 +88,49 @@ class vswitch : public wf::plugin_interface_t
                 return false;
             }
         });
+
+        std::vector<std::string> workspace_numbers;
+        const std::string select_prefix = "select_workspace_";
+        auto section = wf::get_core().config.get_section("vswitch");
+        for (auto binding : section->get_registered_options())
```

Comment:
Isn't it simpler to iterate over all available workspaces and then query the config for the appropriate option, instead of trying to parse options?

Reply:
Yes, I just copied the principle from expo.